libs/redisconsumer/consumer.go

```diff
@@ -165,15 +167,20 @@ func (redisClient *RedisClient) ReadAndHandle(ctx context.Context, stream, consu
 				logger = &tmp
 				ctx = logger.WithContext(ctx)
 
-				err := handle(ctx, stream, messageID, []byte(sData))
-				if err != nil {
-					if !strings.Contains(err.Error(), "retry-after") {
-						logger.Warn().Err(err).Msg("message handler returned an error")
+				wg.Add(1)
+				go func() {
```

Comment:
Is this another way to handle error propagation from workers instead of using [`errgroup`](https://pkg.go.dev/golang.org/x/sync/errgroup)?